src/components/HomePage.js

```diff
@@ -0,0 +1,36 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { v4 as uuid } from 'uuid';
+import fetchData from '../api/api';
+import { fetchCountriesStatus, getCountries, getCountriesFailure } from '../redux/homePage/homePage';
+import Country from './Country';
+
+const HomePage = () => {
+  const dispatch = useDispatch();
+  const countriesData = useSelector((state) => state.countries.onSuccess);
+  useEffect(async () => {
+    if (!countriesData.length) {
+      try {
+        dispatch(fetchCountriesStatus({ bool: true }));
+        const { data: { response } } = await fetchData('countries');
+        dispatch(fetchCountriesStatus({ bool: false }));
+        dispatch(getCountries(response));
+      } catch (err) {
+        dispatch(getCountriesFailure({ err }));
+      }
+    }
+  }, []);
+
+  return (
+    <>
+      <h2 className="font-gill bg-blue-800 text-white text-sm  pl-2">Leagues by Countries</h2>
+      <div className="grid grid-cols-2 h-full bg-blue-700">
+        {countriesData?.map((country) => (
+          <Country key={uuid()} country={country} />
+        ))}
+      </div>
+    </>
+  );
+};
```

Comment:
- [ ] It would be great to implement in the top of the home component some images or something that represent the app. As you could see in the next screenshots there are some icons that would make the app looks more professional. Please Stick to the design guidelines as much as possible.

- [ ] The Home page has a list of categories that could be filtered using given parameters, e.g. category name (Search Bar)

![Screen Shot 2022-02-01 at 20 31 29](https://user-images.githubusercontent.com/67211919/152085421-5171bb28-ec94-428a-bcb4-8f3b03128768.png)